war/src/main/js/templates/plugin-manager/available.hbs

```diff
@@ -65,12 +65,12 @@
                 </div>
             {{/if}}
         </td>
-        <td width="25%">
-            {{#if this.releaseTimestamp }}
+        {{#if this.releaseTimestamp }}
+            <td width="25%" data="{{ this.releaseTimestamp.iso8601 }}">
                 <time datetime="{{ this.releaseTimestamp.iso8601 }}">
                     {{ this.releaseTimestamp.displayValue }}
                 </time>
-            {{/if}}
-        </td>
+            </td>
+        {{/if}}
```

Comment:
I'm not an expert, but I worry that if there is no release `timestamp` that this row would not have the expected number of columns and cause other problems. Perhaps it might be safer to limit the scope of this change to the addition of the new `data` attribute?

For example …

```html
        <td width="25%"{{#if this.releaseTimestamp }} data="{{ this.releaseTimestamp.iso8601 }}"{{/if}}>
            {{#if this.releaseTimestamp }}
                <time datetime="{{ this.releaseTimestamp.iso8601 }}">
                    {{ this.releaseTimestamp.displayValue }}
                </time>
            {{/if}}
        </td>
```

… or …

```html
        {{#if this.releaseTimestamp }}
            <td width="25%" data="{{ this.releaseTimestamp.iso8601 }}">
                <time datetime="{{ this.releaseTimestamp.iso8601 }}">
                    {{ this.releaseTimestamp.displayValue }}
                </time>
            </td>
        {{else}}
            <td width="25%"></td>
        {{/if}}
```

If the experts disagree with me, feel free to disregard this comment.

Reply:
I like the second option better. It's more readable.

Reply:
> That was also worrying me, but i wasn't able to track any plugin without this parameter.

That doesn't mean we should not consider that case. We should consider, if only because the existing code does.